backgroundJobs/priceFetcher.js

```diff
@@ -239,6 +239,14 @@ const specialAssetPriceFetchers = {
     const {data} = await retry(axios.get, [coingeckoCall])
     const apiPrice = Object.values(data)[0].usd || 0
     return apiPrice
+  },
+  CRO_0x87EFB3ec1576Dec8ED47e58B832bEdCd86eE186e: async (web3, network, address) => {
+    // fetch TUSD price from coingecko simple price API
+    const coingeckoCall = "https://api.coingecko.com/api/v3/simple/token_price/ethereum?contract_addresses=0x0000000000085d4780B73119b644AE5ecd22b376&vs_currencies=USD"
```

Comment:
Would it be more readable to fetch TUSD price from `https://api.coingecko.com/api/v3/simple/price?ids=true-usd&vs_currencies=USD`?

Reply:
yes, probably. i just copied your code for that. but it is a great idea. Would you fix it?